src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -52,7 +55,12 @@ class PressableWithSecondaryInteraction extends Component {
                     this.props.onSecondaryInteraction(e);
                 }}
                 onPressOut={this.props.onPressOut}
-                onPress={this.props.onPress}
+                onPress={(e) => {
+                    if (this.props.withoutFocus && this.pressableRef) {
```

Comment:
Please add the comment explaining why this is needed
```
/**
 * This component prevents the tapped element from capturing focus.
 * We need to blur this element when clicked as it opens modal that implements focus-trapping.
 * When the modal is closed it focuses back to the last active element.
 * Therefore it shifts the element to bring it back to focus.
 * https://github.com/Expensify/App/issues/6806
 * https://github.com/Expensify/App/issues/14148
 */
 ```
 And I think `PressableWithoutFocus` can be deleted, since we're not using it anymore

Reply:
Added the comment as well.